recommender/commands/train.py

```diff
@@ -0,0 +1,49 @@
+# pylint: disable=missing-function-docstring
+
+"""Flask CLI train commands"""
+
+from recommender.engines.autoencoders.inference.embedding_component import (
+    EmbeddingComponent,
+)
+from recommender.engines.autoencoders.training.pipeline import AEPipeline
+from recommender.engines.ncf.training.pipeline import NCFPipeline
+from recommender.engines.rl.training.pipeline import RLPipeline
+from recommender.pipeline_configs import (
+    AUTOENCODERS_PIPELINE_CONFIG,
+    NCF_PIPELINE_CONFIG,
+    RL_PIPELINE_CONFIG_V1,
+    RL_PIPELINE_CONFIG_V2,
+)
+
+
+def _ae():
+    """Run AutoEncoders pipeline"""
+    AEPipeline(AUTOENCODERS_PIPELINE_CONFIG)()
+
+
+def _embedding():
+    """Embed User and Services and save dense_tensors to cache"""
+    EmbeddingComponent()()
+
+
+def _ncf():
+    """Run NCF pipeline"""
+    NCFPipeline(NCF_PIPELINE_CONFIG)()
+
+
+def _rl():
+    """Run RL (TD3) pipeline"""
+    RLPipeline(RL_PIPELINE_CONFIG_V1)()
```

Comment:
Should there be also a function that call only RLPipeline with version RL_PIPELINE_CONFIG_V2, shouldn't?